src/WtsApi32.Desktop/WtsApi32+WtsSafeServerHandle.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Exported functions from the WtsApi32.dll Windows library
+    /// that are available to Desktop apps only.
+    /// </content>
+    public static partial class WtsApi32
+    {
+        /// <summary>
+        /// Represents a Wts server handle that can be closed with <see cref="WTSCloseServer(IntPtr)"/>.
+        /// </summary>
+        public class WtsSafeServerHandle : SafeHandle
```

Comment:
I would prefer always use the prefix `Safe..`, so I would suggest `SafeWtsServerHandle` or `SafeTerminalServerHandle` which more descriptive.